pepper-apis/src/test/java/org/broadinstitute/ddp/event/pubsubtask/PubSubTaskTestUtil.java

```diff
@@ -0,0 +1,97 @@
+package org.broadinstitute.ddp.event.pubsubtask;
+
+import static org.broadinstitute.ddp.event.pubsubtask.api.PubSubTask.ATTR_TASK_TYPE;
+import static org.broadinstitute.ddp.event.pubsubtask.impl.updateprofile.UpdateProfileConstants.ATTR_PARTICIPANT_GUID;
+import static org.broadinstitute.ddp.event.pubsubtask.impl.updateprofile.UpdateProfileConstants.ATTR_STUDY_GUID;
+import static org.broadinstitute.ddp.event.pubsubtask.impl.updateprofile.UpdateProfileConstants.ATTR_USER_ID;
+import static org.broadinstitute.ddp.event.pubsubtask.impl.updateprofile.UpdateProfileConstants.TASK_TYPE__UPDATE_PROFILE;
+
+
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import org.broadinstitute.ddp.event.pubsubtask.api.PubSubTask;
+import org.broadinstitute.ddp.event.pubsubtask.api.PubSubTaskProcessorFactoryAbstract;
+import org.broadinstitute.ddp.event.pubsubtask.api.PubSubTaskResult;
+import org.broadinstitute.ddp.event.pubsubtask.api.ResultSender;
+import org.broadinstitute.ddp.event.pubsubtask.impl.updateprofile.UpdateProfileProcessor;
+
+public class PubSubTaskTestUtil {
+
+    public static final String PROJECT_ID = "projectId";
+    public static final String PUBSUB_SUBSCRIPTION = "pubSubSubscription";
+
+    public static final String TEST_MESSAGE_ID = "msg_id";
+    public static final String TEST_PARTICIPANT_GUID = "participant_guid";
+    public static final String TEST_USER_ID = "user_id";
+    public static final String TEST_STUDY_GUID = "study_guid";
+
+    public static final String EMAIL = "email";
+    public static final String EDUCATION = "education";
+    public static final String MARITAL_STATUS = "maritalStatus";
+    public static final String FIRST_NAME = "firstName";
+    public static final String LAST_NAME = "lastName";
+
+    public static final String TEST_EMAIL = "test@broadinstitute.org";
```

Comment:
I recommend `test@datadonationplatform.org`. You never know, someone at Broad might actually have the email `test@broad`!

Reply:
Fixed, thanks